kahuna/public/js/main.js

```diff
@@ -220,10 +218,24 @@ kahuna.run(['$rootScope', 'httpErrors',
             function($rootScope, httpErrors) {
 
     $rootScope.$on('events:error:unauthorised', () =>
-        $rootScope.$emit('track:event', 'Authentication', null, 'Error', null, { 'Error code': httpErrors.unauthorised.errorCode }));
+        $rootScope.$emit(
```

Comment:
minor: this could be its own pr...

Reply:
Linting wouldn't pass without it :(.

It's a separate commit at least, so it should be easy to unpick if needed?

Reply:
Sorry, I hadn't realised it was a whitespace change! Ignore me entirely!